notes/affiliates.md

```diff
@@ -0,0 +1,100 @@
+# Our Affiliate Initiatives
+
+There are a lot of related initiatives in the R infrastructure & cross-pharma
+space aiming to deliver open solutions to health authority engagement. 
+
+This document aims to outline some of these intiatives and help to outline the
+overlap and differences between initiatives:
+
+## [the R Consortium](https://www.r-consortium.org/)
+
+> The central mission of the R Consortium is to work with and provide support to
+> the R Foundation and to the key organizations developing, maintaining,
+> distributing and using R software through the identification, development and
+> implementation of infrastructure projects.
+
+The R Consortium sponsors many working groups, such as the R Validation Hub. The
+R Consortium represents general R community interests, which includes supporting
+adoption of R in industry.
+
+Noteable initiatives sponsored by the R Consortium that will be helpful partners
+to help us tackle regulatory-ready package assessment and/or distribution
+include:
+
+- the [Submissions Working Group](https://github.com/RConsortium/submissions-wg)
+- the [R Repositories Working Group](https://github.com/RConsortium/r-repositories-wg)
+
+## [the R Validation Hub](https://www.pharmar.org/)
+
+
+> The main challenge of using R in late-phase trials is ensuring validation
+> documentation. In June 2018, the R-Consortium awarded funding to create an
+> online repository for R package validation in accordance with regulatory
+> standards.
+>
+> > R Validation Hub is a collaboration to support the adoption of R within a
+> > biopharmaceutical regulatory setting
+
+The R Validation Hub is our parent organization. We are one of multiple
+initiatives that each support the grand vision of streamlining the use of R in
+a regulated setting.
+
+In the past this has involved endorsing a white paper that details how R might
+be used in a regulated setting, building tools to support individual R package
+evaluation, and offering an open forum for companies to share their approaches.
+
+To further support this, the next phase of the R Validation Hub initiatives have
+been focused on lifting these activities out of individual organizations and
+into an open, shared context. This is where we come in!
```

Comment:
@dgkf do we have anyone from GSK onboard already?

Reply:
I don't see anyone in any of our invites with a GSK email. I don't think we have anyone whose reached out yet.